grommet/grommet
# [TS] Migrate Main test to tsx

#### What does this PR do?

This PR migrates Main component tests from `.js` to `.tsx`.

#### Where should the reviewer start?

- Make sure all tests pass
- Review background context section below to see what was changed and why.

#### What testing has been done on this PR?

- Ran tests manually and passed tests locally
- CircleCI Tests passed 🎉 

#### How should this be manually tested?

- The reviewer can review and verify that the tests run properly. There were no actual changes made to the tests other than the renaming of the file.

#### Any background context you want to provide?

Upon switch to `.tsx` and initial run of `ts-jest` I got the following output:

Input: `yarn test Main`
Output:
```
yarn run v1.22.10
warning ../package.json: No license field
$ jest --runInBand Main
 PASS  src/js/components/Main/__tests__/Main-test.tsx (27.958 s)
  Main
    ✓ should have no accessibility violations (158 ms)

Jest: "global" coverage threshold for statements (86.9%) not met: 17.64%
Jest: "global" coverage threshold for branches (76.3%) not met: 2.54%
Jest: "global" coverage threshold for lines (87.9%) not met: 18.57%
Jest: "global" coverage threshold for functions (88%) not met: 4.93%
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   1 passed, 1 total
Time:        29.699 s
Ran all test suites matching /Main/i.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Because all test cases passed without issue, I didn't have to make any changes to the tests. 

#### What are the relevant issues?

See #5599

#### Do the grommet docs need to be updated?

No

#### Should this PR be mentioned in the release notes?

No

#### Is this change backwards compatible or is it a breaking change?

Backwards compatible.
